webapi/src/main/scala/org/knora/webapi/responders/v1/ProjectsResponderV1.scala

```diff
@@ -477,17 +443,28 @@ final case class ProjectsResponderV1(actorDeps: ActorDeps) extends Responder(act
    * @param shortcode the shortcode of the project.
    * @return a [[Boolean]].
    */
-  def projectByShortcodeExists(shortcode: String): Future[Boolean] =
+  def projectByShortcodeExists(shortcode: String): Task[Boolean] =
     for {
-      askString <- Future(
+      askString <- ZIO.attempt(
                      org.knora.webapi.messages.twirl.queries.sparql.admin.txt
                        .checkProjectExistsByShortcode(shortcode = shortcode)
                        .toString
                    )
-      // _ = log.debug("projectExists - query: {}", askString)
-
-      checkProjectExistsResponse <- appActor.ask(SparqlAskRequest(askString)).mapTo[SparqlAskResponse]
+      checkProjectExistsResponse <- triplestoreService.sparqlHttpAsk(askString)
       result                      = checkProjectExistsResponse.result
 
     } yield result
 }
+object ProjectsResponderV1Live {
+  val layer: URLayer[
+    StringFormatter with TriplestoreService with MessageRelay with IriService with AppConfig,
```

Comment:
```suggestion
    StringFormatter with TriplestoreService with MessageRelay,
```

Reply:
these could probably be removed as well?

Reply:
Thanks.